src/components/accordion/accordion.css

```diff
@@ -44,6 +44,10 @@
   font-size: inherit;
 }
 
+.accordion-body p:first-child {
```

Comment:
Skulle dette være first-child generelt i stedet for kun _paragraph_? Det ser ud til, at vi godt kan komme ud for tilfælde, hvor first-child feks er et anchor element (taget udgangspunkt i footer design, med accordions som heldigvis ikke blev til noget)

Reply:
det kun p tags der har margin by default. så :first-child generelt vil overskrive hvis amn måsker en div med margin-top på

Reply:
Super, giver mening